generate_csv.py

```diff
@@ -21,7 +22,7 @@ def compute_hash_and_length(file_path):
         return None, None
 
 
-def generate_csv(mirror_dir, output_csv):
+def generate_all_csv(mirror_dir, output_csv):
```

Comment:
Proposal: have `generate_all_csv` call `generate_updated_csv` as a helper. The former would just walk thru all the simple folders to build `updated_pkg` to pass to the latter.